packages/ng/forms/checkbox-input/checkbox-input.component.ts

```diff
@@ -23,14 +23,16 @@ let nextId = 0;
 		},
 	],
 	host: {
-		class: 'checkboxField',
+		'[class.checkboxField]': 'true',
+		'[class.mod-checklist]': 'isChecklist()',
 	},
 })
 export class CheckboxInputComponent implements FilterPillInputComponent {
 	formField = inject<FormFieldComponent>(FORM_FIELD_INSTANCE, { optional: true });
 
 	isFilterPill = false;
 	filterPillInputId = `lu-checkbox-pill-input-${nextId++}`;
+	isChecklist = input<boolean, boolean>(false, { transform: booleanAttribute });
```

Comment:
```suggestion
	isChecklist = input(false, { transform: booleanAttribute });
```
We need to let TS infer the type else it won't work as a flag and will require a boolean value.